acprep

```diff
@@ -510,7 +500,7 @@ class PrepareBuild(CommandLineApp):
 
         self.log.info("Installing Ledger's build dependencies ...")
 
-        system = str(self.get_stdout('uname', '-s'))
+        system = self.get_stdout('uname', '-s').decode('utf8')
```

Comment:
It seems you're undoing the change from the other commit there?

Reply:
yes that was a mistake, and I did not clean the change history.